reddit_decider/__init__.py

```diff
@@ -758,10 +736,7 @@ def get_float(self, feature_name: str, default: float = 0.0) -> float:
 
         :return: the float value of the dyanimc config if it is active/exists, :code:`default` parameter otherwise.
         """
-        decider = self._get_decider()
-        if not decider:
-            return default
-        return self._get_dynamic_config_value(feature_name, decider.get_float, default)
+        return self._get_dynamic_config_value(feature_name, default, [float, int])
```

Comment:
Why are floats special? We're pretty explicit about the DC types.

Realistically both `int` and `float` should be replaced with `number` (implemented as a float) since that's what adheres to the JSON spec. But that's a bigger change. However I'm not sure why this is permissive and ints aren't.

Reply:
this was to assuage [these tests](https://github.com/reddit/experiments.py/blob/299ae13d3618217e48bfcc4772e3c184b6ffb9b3/tests/decider_tests.py#L1409-L1429) but after doing you suggestion of casting via `dc_type()`, all this nonsense goes away : )